sdk/communication/azure-communication-networktraversal/src/test/java/com/azure/communication/networktraversal/CommunicationRelayAsyncTests.java

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.networktraversal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import com.azure.communication.common.*;
+import com.azure.communication.identity.*;
+import com.azure.communication.networktraversal.models.CommunicationRelayConfiguration;
+import com.azure.communication.networktraversal.models.CommunicationIceServer;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.rest.Response;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+public class CommunicationRelayAsyncTests extends CommunicationRelayClientTestBase {
+    private CommunicationRelayAsyncClient asyncClient;
+
+    @ParameterizedTest
+    @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    public void createRelayClientUsingManagedIdentity(HttpClient httpClient) {
+
+        // Arrange
+        CommunicationRelayClientBuilder builder = createClientBuilderUsingManagedIdentity(httpClient);
```

Comment:
Are these live tests? Where are the recordings for these tests?

Reply:
Added them :)